lib/apipie_bindings/api.rb

```diff
@@ -224,7 +224,7 @@ def http_call(http_method, path, params={}, headers={}, options={})
         rescue => e
           clear_credentials if e.is_a? RestClient::Unauthorized
           log.debug e.message + "\n" +
-            inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)
+            (inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)) rescue ''
```

Comment:
> My point is, this change would still prevent logging the original error message.

What exactly do you want to file into logs when the communication was
terminated (the connection was perhaps closed by peer). There's no data.
That's the reason why it fails.

The goal of this patch really is to prevent the rescue block from
erroring out, that's all.

-- 
Later,
 Lukas #lzap Zapletal